prowler/providers/aws/services/eks/eks_cluster_private_nodes_enabled/eks_cluster_private_nodes_enabled.metadata.json

```diff
@@ -0,0 +1,36 @@
+{
+  "Provider": "aws",
+  "CheckID": "eks_cluster_private_nodes_enabled",
+  "CheckTitle": "Ensure Clusters are Created with Private Nodes",
+  "CheckType": [
+    "Security",
+    "Configuration"
+  ],
+  "ServiceName": "EKS",
+  "SubServiceName": "",
+  "ResourceIdTemplate": "",
+  "Severity": "high",
+  "ResourceType": "AwsEksCluster",
+  "Description": "Ensure that clusters are created with private nodes, disabling public IP addresses for cluster nodes. Private nodes have no public IP addresses, restricting access to internal networks and enhancing security.",
+  "Risk": "Without private nodes, cluster nodes may have public IP addresses, increasing the attack surface and exposing them to potential threats from the internet.",
+  "RelatedUrl": "",
+  "Remediation": {
+    "Code": {
+      "CLI": "aws eks update-cluster-config --region region-code --name my-cluster --resources-vpc-config endpointPublicAccess=true,publicAccessCidrs=\"203.0.113.5/32\",endpointPrivateAccess=true",
```

Comment:
Could we leave this just with `aws eks update-cluster-config --region region-code --name my-cluster --resources-vpc-config endpointPrivateAccess=true`?

Reply:
I put the command that appears in the CIS.

Reply:
Added to change it since the CIS is not right.